### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by ehmueller** [trufflepig-forensics/memdump-libvirt#1]
Hi, This is not an issue but only a question. How does this tool differ from running "virsh dump <vmname>" to a file?

thanks
Eric

**Comment by cmueller-tp** [comment]
Hi Eric,
I was actually not aware of that option, seems to be the better way of doing it.

Have you experience using it and how stable it is?

**Comment by ehmueller** [comment]
No, I have not but I was doing research on this matter and came across your project. I wasn’t sure if both methods provided the same result , hence my question.

I have a requirement to do memory dump for a new project so I will be testing the virsh solution. Thanks for the feedback

**Comment by cmueller-tp** [comment]
Ok, can you report back how it is? I'm really not happy with this solution so I'd be happy too to have an alternative.
